### PR TITLE
Development Phase #1

### PR DESCRIPTION
**Pull request by akoushke**
https://wxsd.atlassian.net/browse/FER-1357?filter=10020

### CODE REVIEW

File: src/components/DemoCard/DemoCard.svelte [hunk 1]
```diff
@@ -0,0 +1,143 @@
...
+  export let brandLogo: Data;
+  export let uuid: string;
+
+  let isLoading = name == '+ New Demo +';
+  let isSelected = false;
+  let copyIsLoading = false;
+  let copied = false;
+
+  const switchDemoCard = (e) => {
+    if (e.target.tagName != 'I' && !isLoading) {
```
Comment (akoushke) [inline src/components/DemoCard/DemoCard.svelte:23]: add comment on why `I`

File: src/components/DemoCard/DemoCard.svelte [hunk 2]
```diff
@@ -0,0 +1,143 @@
...
+            </form>
+          </div>
+        </nav>
+      </div>
+    </article>
+  </div>
+</div>
+
+<style>
+  .demo {
```
Comment (akoushke) [inline src/components/DemoCard/DemoCard.svelte:96]: try to condense this

File: src/lib/shared/reactive-url.ts [hunk 3]
```diff
@@ -0,0 +1,37 @@
+/* eslint-disable prefer-rest-params */
```
Comment (akoushke) [inline src/lib/shared/reactive-url.ts:3]: add some comments on why we need this

File: src/routes/api/user/[userId]/demos/index.ts [hunk 4]
```diff
@@ -0,0 +1,50 @@
+import type { RequestEvent } from '@sveltejs/kit';
+import { User } from '../../../../../database/entities';
+import * as entity from '../../../../../database/entities';
+import config from '../../../../../../mikro-orm.config';
+import { LoadStrategy, MikroORM } from '@mikro-orm/core';
+
+export const GET: RequestHandler = async (requestEvent: RequestEvent) => {
```
Comment (akoushke) [inline src/routes/api/user/[userId]/demos/index.ts:9]: clean this up

**Review by tahanson-cisco — APPROVED** [review]
Tested this locally and it looks good - Arash took down a couple of notes for UI inconsistencies with Firefox, but nothing game breaking.

**Pull request merged**